grafonnet-base/veneer/row.libsonnet

```diff
@@ -0,0 +1,12 @@
+local d = import 'github.com/jsonnet-libs/docsonnet/doc-util/main.libsonnet';
+
+function(name, panel)
+  {
+    '#new':: d.func.new(
+      'Creates a new %s panel with a title.' % name,
+      args=[d.arg('title', d.T.string)]
+    ),
+    new(title):
+      self.withTitle(title)
+      + self.withType(),
```

Comment:
why no value in `withType`?

Reply:
It's an implicit `constant` in the schema, this should not be modified as only 1 value is allowed.